aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -98,3 +98,15 @@ impl From<AFeatureFlag> for FeatureFlag {
         }
     }
 }
+
+impl Features {
+    pub(crate) fn has_modified(&self, on_chain_features: &AFeatures) -> bool {
```

Comment:
`has_modified` what's modified?

I look at the code, and it tells me that it's if one of hte feature flags provided is different than the current state.  Could you add a comment for this?